cluster-autoscaler/nodegroup/spec.go

```diff
@@ -0,0 +1,69 @@
+package nodegroup
```

Comment:
please keep it in the config/dynamic package. No reason for creating another top-level package.